pkg/morph/client/constructor.go

```diff
@@ -250,6 +259,28 @@ func newActor(ws *rpcclient.WSClient, acc *wallet.Account, cfg cfg) (*actor.Acto
 	}})
 }
 
+func newProxyActor(ws *rpcclient.WSClient, proxy util.Uint160, acc *wallet.Account, cfg cfg) (*actor.Actor, error) {
+	return actor.New(ws, []actor.SignerAccount{
+		{
+			Signer: transaction.Signer{
+				Account: proxy,
+				Scopes:  transaction.None,
+			},
+			Account: wallet.NewContractAccount(proxy),
+		},
+		{
+			Signer: transaction.Signer{
```

Comment:
Do we need this signer at all?

Reply:
i thought as a chain guy you would support signing an operation by a real sender. in practice, no, we do not need it

Reply:
I don't support useless signatures. We have enough of them already here.

Reply:
ok